tools/wpt/browser.py

```diff
@@ -575,7 +587,26 @@ def chromium_platform_string(self):
 
         return platform
 
+    def _chromium_package_name(self):
+        return "chrome-%s" % self.platforms.get(uname[0]).lower()
+
+    def _latest_chromium_snapshot_url(self):
+        # Make sure we use the same revision in an invocation.
+        if self._last_change is None:
+            arch = self._chromium_platform_string()
```

Comment:
nit: what is `arch` short for? archive? architecture? Wonder if another word would be clearer.

Reply:
It's architecture. I'll change the code.